src/io.c

```diff
@@ -102,6 +102,11 @@ void* wolfSSH_GetIOWriteCtx(WOLFSSH* ssh)
     return NULL;
 }
 
+void wolfSSH_SetReqSuccess(WOLFSSH_CTX *ctx, WS_CallbackReqSuccess cb)
```

Comment:
Can you move this over to ssh.c, please? All the other callbacks are there, except for the I/O callbacks in io.c. The I/O callbacks are here because there will be several default callbacks for various OS.

Reply:
Moved them to ssh.c